api/registry/test/test_passport_get_stamps.py

```diff
@@ -0,0 +1,197 @@
+from urllib.parse import urlparse
+
+import pytest
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+from django.test import Client
+from registry.models import Passport, Stamp
+from web3 import Web3
+
+User = get_user_model()
+web3 = Web3()
+web3.eth.account.enable_unaudited_hdwallet_features()
+my_mnemonic = settings.TEST_MNEMONIC
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def paginated_stamps(scorer_community, passport_holder_addresses):
+    passport = Passport.objects.create(
+        address=passport_holder_addresses[0]["address"],
+        community=scorer_community,
+    )
+
+    stamps = []
+
+    for i in range(6):
+        stamp = Stamp.objects.create(
```

Comment:
I would create 10 stamps for testing, and here is one more case to cover.
If we would read with `limit-2`, and always follow the next link, we would get these items:
      - [10 9]
      - [8, 7]
      - [6,5]
      - [4,3]
      - [2,1]
      
New test:
- let us read until we get back the batch [6, 5]
- read the batch from the prev link, and ensure it is [8, 7]

The risk I want to cover: if one would not sort the elements properly in the query, he would get back the wrong result.